libs/deso-protocol/src/lib/identity/Identity.ts

```diff
@@ -418,6 +421,36 @@ export class Identity {
       this.transactions
     );
   }
+  public async messagingGroups(
+    publicKeyBase58Check: string
+  ): Promise<MessagingGroupPayload> {
+    if (this.host === 'server') throw Error(SERVER_ERROR);
+
+    if (!this.storageGranted) {
+      await this.guardFeatureSupport();
+    }
+
+    const prompt = requestMessagingGroups(
+      this.getUri(),
+      this.isTestnet(),
+      undefined,
+      {
+        operation: MessagingGroupOperation.DEFAULT_KEY,
+        applicationMessagingPublicKeyBase58Check: publicKeyBase58Check,
```

Comment:
this should be a unique parameter. Kinda pointless to encrypt the messaging private key back to oneself.